test/patch_wordpress_test.js

```diff
@@ -29,4 +30,84 @@ describe('grunt_patch_wordpress', function () {
 			done()
 		})
 	})
+
+	describe( 'map_old_to_new_file_path', function() {
+		var file_mappings = {
+			'src/wp-admin/js/color-picker.js': 'src/js/_enqueues/lib/color-picker.js'
+		};
+
+		it ( 'replaces old file paths with new file paths in the diff', function() {
+			grunt.file.copy( "./test/fixtures/patch_wordpress_1.diff", "./test/fixtures/patch_wordpress_1_copy.diff" );
```

Comment:
We should add a `test/temp` folder whose content (outside of a keeper file) is where we put these temp files. It will help keep the fixtures folder clean.